makefile

```diff
@@ -37,20 +37,20 @@ docker_volume_rm:
 docker_volume_recreate: docker_volume_rm docker_volume_create
 
 docker_up_postgres:
-	BAG3D_DOCKER_IMAGE_TAG=$(BAG3D_DOCKER_IMAGE_TAG) docker compose -p $(COMPOSE_PROJECT_NAME) -f docker/compose.yaml up -d data-postgresql
+	export SSH_KEY="$$(cat ~/.ssh/id_rsa)" ; BAG3D_DOCKER_IMAGE_TAG=$(BAG3D_DOCKER_IMAGE_TAG) docker compose -p $(COMPOSE_PROJECT_NAME) -f docker/compose.yaml up -d data-postgresql
```

Comment:
this is the way I found to safely transfer the ssh key in the image - under the assumption that the pipeline would be run under "dagster" user. If there is a better way, let me know.

Reply:
This copies the host user's private key into an environment variable and also it builds the ssh key into the docker image, a big no no. We need to mount the host ssh key into the container when running it.

Reply:
ΟΚ, clear, I thought that since it is in our server it should not be an issue. I will find another way!